plugins/Goals/lang/en.json

```
@@ -3,63 +3,63 @@
         "Contains": "contains %s",
-        "ConversionByTypeReportDocumentation": "This report provides detailed information about the goal performance (conversions, conversion rates and revenue per visit) for each of the categories available in the left panel. %1$s Please click on one of the categories to view the report. %2$s For more information, read the %3$sTracking Goals documentation%4$s",
+        "ConversionByTypeReportDocumentation": "This report provides detailed info about the goal performance (conversions, conversion rates and revenue per visit) for each of the categories available in the left panel. %1$s Please click on one of the categories to view the report. %2$s For more info, read the %3$sTracking goals documentation%4$s",
         "ConversionRate": "%s conversion rate",
         "ConversionRatePageViewedBefore": "Viewed before %s rate",
         "Conversions": "%s conversions",
         "ConversionsDescription": "conversions",
-        "ConversionsOverview": "Conversions Overview",
-        "ConversionsOverviewBy": "Conversions overview by type of visit",
+        "ConversionsOverview": "Conversion Overview",
+        "ConversionsOverviewBy": "Conversion overview by type of visit",
         "DaysToConv": "Days to Conversion",
-        "DaysToConvReportDocumentation": "This report shows how many days pass before your visitors convert a goal.",
+        "DaysToConvReportDocumentation": "This report shows how many days pass on average before your visitors convert a goal.",
         "Details": "Goal details",
         "DefaultGoalConvertedOncePerVisit": "(default) Goal can only be converted once per visit",
-        "DefaultRevenueLabel": "Goal default revenue",
-        "DefaultRevenueHelp": "For example, a Contact Form submitted by a visitor may be worth $10 on average. Matomo will help you understand how well your visitors segments are performing.",
-        "DeleteGoalConfirm": "Are you sure you want to delete the Goal %s?",
+        "DefaultRevenueLabel": "Default goal revenue",
+        "DefaultRevenueHelp": "For example, a contact form submitted by a visitor may be worth $10 on average. Matomo will help you understand how well your visitor segments are performing.",
+        "DeleteGoalConfirm": "Delete the %s goal?",
         "DocumentationRevenueGeneratedByProductSales": "Product sales. Excludes tax, shipping and discount",
         "Download": "Download a file",
-        "Ecommerce": "Ecommerce",
-        "EcommerceAndGoalsMenu": "Ecommerce & Goals",
-        "EcommerceLog": "Ecommerce Log",
-        "EcommerceOrder": "Ecommerce order",
-        "EcommerceOverview": "Ecommerce Overview",
-        "EcommerceReports": "Ecommerce Reports",
+        "Ecommerce": "E-commerce",
+        "EcommerceAndGoalsMenu": "E-commerce and Goals",
```

```suggestion
        "EcommerceAndGoalsMenu": "e-commerce and goals,",
```
As per https://github.com/matomo-org/matomo/pull/19307/files#r894220850